src/CofreeBot/Bot.hs

```diff
@@ -232,6 +136,11 @@ fixBot (Bot b) = go
   go :: s -> Behavior m i o
   go s = Behavior $ \i -> second go <$> b s i
 
+-- | Batch process a list of inputs @i@ with a single 'Behavior',
+-- interleaving the effects, and collecting the resulting outputs @o@.
```

Comment:
The wording is a bit odd here, the effects due to each input remain grouped together, there isn't any interleaving going on (or maybe I'm not quite understanding what interleaving means in this context).